auraed/tests/cells_list.rs

```diff
@@ -0,0 +1,113 @@
+use client::cells::cell_service::CellServiceClient;
+use common::cells::CellServiceAllocateRequestBuilder;
+use iter_tools::Itertools;
+use proto::cells::{CellGraphNode, CellServiceListRequest};
+use test_helpers::*;
+
+mod common;
+
+#[test_helpers_macros::shared_runtime_test]
+async fn cells_list_must_list_allocated_cells_recursively() {
```

Comment:
Did you accidentally put this test here instead of the test that was initially in `e2e-tests/src/cells_list.rs`? IMO we should remove this test as it doesn't really belong here (if we consider the tests in this folder e2e tests). Also, everything this test is testing (and more) was already covered in the above-mentioned e2e test.

FWIW I am introducing a lot more unit-style testing in my upcoming PR to move us to a healthier pyramid shape

Reply:
I'll revert